Enterprise-CMCS/eAPD
# Tbolt/4416 mmis proposed budget

Resolves #4416 

### Description
* Updates the budget calculation in the backend to compute key state personnel for MMIS apds
* Adds new selector for calculating the key state personnel totals to support new table in Proposed Budget for MMIS apds
* Adds new component to display new key state personnel table for MMIS apds
* Updates to Proposed Budget page language for MMIS apds
* Updates Proposed Budget page to match Figma design

### Significant changes or possible side effects
* HITECH apds should work as it did prior to these code changes. It's possible the MMIS changes could have impacted HITECH apds

### Steps to manually verify this change

1. Login as a user who can view APDs
2. Open a HITECH apd
3. Verify that Proposed Budget page for HITECH is working as intended. All tables are still present, totals, subtotals, cost categories, export view, etc...
4. Navigate to or create a MMIS apd
5. Verify that Proposed Budget page now loads
6. Verify that all acceptance criteria in the original ticket is working
7. Verify the Proposed Budget page matches the Figma file (with the exception of the newly added Medicaid Share % that was discussed in standup)

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
